react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import { FormControlLabel, makeStyles } from "@material-ui/core";
+import Switch from "@material-ui/core/Switch";
```

Comment:
You wrote over my change. You should avoid force-pushing.

The second import statement can be combined with the first.